src/hooks/useLocaleFormatter/useLocaleFormatter.ts

```diff
@@ -89,7 +89,7 @@ const getParts = (locale: string, fiatType = 'USD') => {
       style: 'currency',
       useGrouping: true,
     })
-    const parts = groupFormatter.formatToParts(1234567.1234567890123456789)
+    const parts = groupFormatter.formatToParts(1234567)
```

Comment:
Not sure to understand why you did this, any rational to share?

Reply:
Lint is failing on this guy (unsafe float) and as far as I can see, this isn't actually surfaced to users, just some legacy fluff copied and pasted from web. `formatToParts` is supposed to take an actual amount, which confirms the assumption.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/formatToParts